src/components/ScrollView/index.tsx

```diff
@@ -1,5 +1,5 @@
 import styled from '@emotion/styled'
-import React from 'react'
+import React, { FunctionComponent } from 'react'
 import Box from '../Box'
 
 const StyledScrollView = styled(Box)`
```

Comment:
We never use xstyled system on this component, can we piggy back the change on this PR ? 

```suggestion
const StyledScrollView = styled.div`
```
```suggestion
const StyledScrollView = styled(Box)`
```